### PR TITLE
Consolidate style attributes in JS files into LESS files

### PR DESCRIPTION
**Pull request by PaulMEdwards**
There were multiple methods of defining styles used in the application.

This PR continues the work started in PR #71 of consolidating all of the various styles into Ant Design's LESS mechanism, doing so for the style attributes within individual JS files.

 [Consolidate ALL styling mechanisms into LESS files](https://trello.com/c/quoDQaOZ/46-consolidate-all-styling-mechanisms-into-less-files)

### CODE REVIEW

**Comment by PaulMEdwards** [comment]
[Consolidate ALL styling mechanisms into LESS files](https://trello.com/c/quoDQaOZ/46-consolidate-all-styling-mechanisms-into-less-files)

File: src/less/index.less [hunk 1]
```diff
@@ -1,34 +1,320 @@
...
+        &-380 {
+            min-height: 380;
+        }
+    }
+}
+
+.max {
+    &-height {
+        &-40vh {
+            max-height: 40hv;
```
Comment (Brodt7258) [inline src/less/index.less:263]: just need to flip the units. `vh` instead of `hv`.

**Review by Brodt7258 — CHANGES_REQUESTED** [review]
tested locally. the only thing that appeared out of place was the height of the SearchField. looks like just a typo in the height units.

File: src/less/index.less (hunk 1 above)
Comment (PaulMEdwards) [reply src/less/index.less]: Derp; thanks @Brodt7258!

**Review by Brodt7258 — APPROVED** [review]
typo fixed, pr approved

**Review by danielterry2121 — APPROVED** [review]
Look's clean as always.

**Pull request merged**